presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergPageSink.java

```diff
@@ -369,6 +370,10 @@ private static Object getIcebergValue(Block block, int position, Type type)
             long timestamp = type.getLong(block, position);
             return ((TimestampType) type).getPrecision() == MILLISECONDS ? MILLISECONDS.toMicros(timestamp) : timestamp;
         }
+        if (type instanceof TimeType) {
+            long time = type.getLong(block, position);
+            return MILLISECONDS.toMicros(time);
```

Comment:
I noticed that in the following Parquet interaction, you used `TIME_MICROS`, but here, you used `TimeType`. Do you want to keep them consistent? (e.g. adding a conversion in the `TypeConverter`?)

Reply:
@ChunxuTang Thanks for you review. Could you describe a little more about `keeping them consistent`? As I understand, we have already converted presto TimeType to iceberg TimeType and vice versa in `TypeConverter`. The code here is just used to convert the value, as presto engine explicitly parse and maintain TimeType value in milliseconds, but iceberg would handle it in microseconds following the spec. After that, at the lower layer, when iceberg interacts with parquet, it uses TIME_MICROS other than TIME_MILLIS.

Please let me know if anything missed.